site/content/upgrading-to-v4.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Guide on upgrading to v4 and above
+docs: DOCS-000
+title: Guild on Upgrading to v4 and above
```

Comment:
```suggestion
title: Guide on upgrading to NGINx Ingress Controller 4.x.x from 3.x.x
```

Reply:
Do we need to say F5 NGINX Ingress Controller in this context?

Reply:
@ADubhlaoich what do yo think?

Reply:
No need to prefix the product name in page titles.

The hard rule is first mention on the body text of a page.

I'm in general agreement that the page title has space to improve.